app/assets/stylesheets/_fills.scss

```diff
@@ -81,4 +85,14 @@ svg{
   &:hover svg, &:active svg{
     fill: color-yiq($body-color);
   }
-}
\ No newline at end of file
+}
+
+.octicon {
+  &.failure, &.error {
+    fill: $notification-closed;
+  }
+  &.pending {
+    fill: $notification-pending;
+    vertical-align: middle;
```

Comment:
alignment should probably be in `_badges.scss`

Reply:
moved to `_badge.sccc`